src/misc/wikipedia-pkcs8.meta

```diff
@@ -0,0 +1,2 @@
+uri = https://en.wikipedia.org/wiki/PKCS_8
+description = Example key from Wikipedia page about PKCS #8
```

Comment:
```suggestion
description = Example key from Wikipedia page about PKCS #8
password = test
```

I noticed those keys are identical, thus just documenting that password suffices.